lms/static/scripts/frontend_apps/components/GroupConfigSelector.tsx

```diff
@@ -9,34 +10,41 @@ import { useUniqueId } from '../utils/hooks';
 import AuthorizationModal from './AuthorizationModal';
 import ErrorModal from './ErrorModal';
 
-/**
- * @typedef {import('../api-types').GroupSet} GroupSet
- */
-
 /**
  * Configuration relating to how students are divided into groups for an
  * assignment.
- *
- * @typedef GroupConfig
- * @prop {boolean} useGroupSet - Whether students are divided into groups based
- *   on a grouping/group set defined in the LMS (the terminology varies depending
- *   on the LMS).
- * @prop {string|null} groupSet - The ID of the grouping to use. This should
- *   be the `id` of a `GroupSet` returned by the LMS backend.
  */
+export type GroupConfig = {
+  /**
+   * Whether students are divided into groups based on a grouping/group set
+   * defined in the LMS (the terminology varies depending on the LMS).
+   */
+  useGroupSet: boolean;
+
+  /**
+   * The ID of the grouping to use. This should be the `id` of a `GroupSet`
+   * returned by the LMS backend.
+   */
+  groupSet: string | null;
+};
+
+type GroupSelectProps = {
+  /** A request for groups is currently in flight */
+  busy: boolean;
```

Comment:
Outside of the scope of these changes, but we might consider being consistent with our conventional `loading` nomenclature for this prop.